solutions/fscloud/README.md

```diff
@@ -0,0 +1,58 @@
+# Profile for IBM Cloud Framework for Financial Services
```

Comment:
This isn't a profile or submodule. Its a DA (Deployable Architecture), so we need to come up with a better title. This will show in the catalog. Will need @SirSpidey input here

Reply:
Same goes for everything we have in the doc here.